src/Guzzle6HttpAdapter.php

```diff
@@ -29,22 +34,27 @@ class Guzzle6HttpAdapter implements HttpAdapter
     private $client;
 
     /**
-     * @param ClientInterface|null $client
+     * @var array Options to pass when sending one or multiple requests with guzzle
      */
-    public function __construct(ClientInterface $client = null)
+    private $options;
+
+    /**
+     * @param ClientInterface|null $client  Guzzle client
+     * @param array                $options Options to pass when sending one or multiple requests with guzzle
+     */
+    public function __construct(ClientInterface $client = null, array $options = [])
     {
-        $this->client = $client ?: new Client();
+        $this->client  = $client ?: new Client();
+        $this->options = $this->buildOptions($options);
     }
 
     /**
      * {@inheritdoc}
      */
-    public function sendRequest(RequestInterface $request, array $options = [])
+    public function sendRequest(RequestInterface $request)
     {
-        $options = $this->buildOptions($options);
-
         try {
-            return $this->client->send($request, $options);
+            return $this->client->send($request, $this->options);
```

Comment:
These options can also be configure in Guzzle at construction time, can't they? If so, we should just wrap the client and forget about options.